ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_SERVERS.yml

```diff
@@ -389,6 +389,33 @@ servers:
           mode: active
           enabled: false
 
+  server18_monitoring_session_source:
+    rack: RackC
+    adapters:
+      - server_ports: [ Eth1 ]
```

Comment:
please use `endpoint_ports`

Reply:
yupyup, the server_ports is the legacy key, as we've talked about :-) 
just had this copied from the existing tests before seeing that its actually legacy